pkg/oc/admin/policy/reconcile_clusterrolebindings_test.go

```diff
@@ -74,108 +68,108 @@ func TestDiffObjectReferenceLists(t *testing.T) {
 
 func TestComputeUpdate(t *testing.T) {
 	tests := map[string]struct {
-		ExpectedBinding *authorizationapi.ClusterRoleBinding
-		ActualBinding   *authorizationapi.ClusterRoleBinding
-		ExcludeSubjects []kapi.ObjectReference
+		ExpectedBinding *rbac.ClusterRoleBinding
+		ActualBinding   *rbac.ClusterRoleBinding
+		ExcludeSubjects []rbac.Subject
 		Union           bool
 
-		ExpectedUpdatedBinding *authorizationapi.ClusterRoleBinding
+		ExpectedUpdatedBinding *rbac.ClusterRoleBinding
 		ExpectedUpdateNeeded   bool
 	}{
 		"match without union": {
-			ExpectedBinding: binding(ref("role"), refs("a")),
-			ActualBinding:   binding(ref("role"), refs("a")),
+			ExpectedBinding: binding("role", subjects("a")),
```

Comment:
I made the ``bindings()`` helper function a bit more friendlier. It now takes a role name and creates the role object internally.